invenio_administration/menu/menu.py

```diff
@@ -62,6 +63,14 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
                     icon=icon,
                 )
 
+    def register_plus_menu_entries(self, current_menu, endpoint):
+        """Register plus menu items to a flask menu instance."""
+        current_menu.submenu("admin_plus.administration").register(
+            f"{endpoint}.oaipmh_create",
+            _('New OAI-PMH set'),
+            order=1,
+        )
+
```

Comment:
I didn't harcode the menu in the end because I thought it was more problematic having to pass the endpoint to the template (in order to avoid hardcoding the URL).